spec/controllers/week_question_controller_spec.rb

```diff
@@ -0,0 +1,20 @@
+require "rails_helper"
+
+RSpec.describe "Weekly Question", type: :request do
+  describe "GET show" do
+    it "returns a 200" do
```

Comment:
Here you can not only change the verb, but also make it more descriptive.

```suggestion
    it "provides the weekly question for the given user" do
```